examples/librispeech/s0/README.md

```diff
@@ -219,3 +219,53 @@ test other
 | LM-tgmed + attention rescoring   | 8.78  | 10.26 |
 | LM-tglarge + attention rescoring | 8.34  | 9.74  |
 | LM-fglarge + attention rescoring | 8.17  | 9.44  |
+
+
+## Efficient Conformer V1 Result
+
+* Feature info: using fbank feature, cmvn, speed perturb, dither
+* Training info: train_u2++_efficonformer_v1.yaml, 8 gpu
+* Decoding info: ctc_weight 0.5, reverse_weight 0.3, average_num 20
+
+test clean
+
+| decoding mode          | full | 18   | 16   |
+|------------------------|------|------|------|
+| attention decoder      | 3.65 | 3.88 | 3.87 |
+| ctc_greedy_search      | 3.46 | 3.79 | 3.77 |
+| ctc prefix beam search | 3.44 | 3.75 | 3.74 |
+| attention rescoring    | 3.17 | 3.44 | 3.41 |
+
+test other
+
+| decoding mode          | full | 18    | 16    |
+|------------------------|------|-------|-------|
+| attention decoder      | 8.51 | 9.24  | 9.25  |
+| ctc_greedy_search      | 8.94 | 10.04 | 10.06 |
+| ctc prefix beam search | 8.91 | 10    | 10.01 |
+| attention rescoring    | 8.21 | 9.25  | 9.25  |
```

Comment:
Thx, the results are much better than the standard conformer! I would suggest adding model params in README for a clearer comparison.

Reply:
OK, no problem